searx/settings.yml

```diff
@@ -953,6 +953,11 @@ engines:
 #    query_fields : '' # query fields
 #    enable_http : True
 
+  - name : springer nature
+    engine : springer
+    shortcut : springer
+    categories : science
+
```

Comment:
I tested some queries of the springer database (e.g. [physics](https://api.springernature.com/metadata/json?&q=physics&api_key=a69685087d07eca9f13db62f65b8f601)), for large results I would recommend a timeout of 6 sec

        timeout : 6

Reply:
Done.